src/infra/core/tests/tests/repos/test_dataset_repository_shared.rs

```diff
@@ -337,8 +337,7 @@ pub async fn test_rename_dataset_same_name_multiple_tenants(repo: &dyn DatasetRe
 }
 
 /////////////////////////////////////////////////////////////////////////////////////////
-
-pub async fn test_rename_dataset_unauthroized(
+pub async fn test_rename_dataset_unauthorized(
```

Comment:
Formatting - keep an empty line after the ////

Reply:
Done

Reply:
https://github.com/kamu-data/kamu-cli/pull/447#discussion_r1455158973